frontend/src/metabase/visualizations/lib/apply_axis.js

```diff
@@ -490,7 +493,8 @@ export function getYValueFormatter(chart, series, yExtent) {
     const metricColumn = series[seriesIndex].data.cols[1];
     const columnSettings = chart.settings.column(metricColumn);
     const columnExtent = options.extent ?? yExtent;
-    const roundedValue = maybeRoundValueToZero(value, columnExtent);
+    const roundedValue =
+      value === null ? "" : maybeRoundValueToZero(value, columnExtent);
```

Comment:
This was necessary to preserve the behavior where nulls display as empty strings.

https://www.deploysentinel.com/ci/runs/64da9f82c4985efd77cda2b9